### ISSUE TITLE
Query Parameters coverage for Workspaces

### ISSUE THREAD

**Issue by rorychatterton** [dahlke/terrasnek#28]
Is it possible to extend the [Workspaces List](https://terrasnek.readthedocs.io/en/latest/workspaces.html#terrasnek.workspaces.TFCWorkspaces.list) endpoint to support query parameters as per the [API documentation](https://www.terraform.io/cloud-docs/api-docs/workspaces#query-parameters). 

It appears to be listed in the [documentation/comments](https://github.com/dahlke/terrasnek/blob/master/terrasnek/workspaces.py#L86), but no obvious way to pass the query as the only supported inputs are: `page, page_size & include`?

I would like to be able to pass through a list like:

```
query = {"search[\"name\"]":"my-awesome-prefix"}
workspaces = tfc_api.workspaces.list_all(query=query)
```

I would imagine the implementation would be similar to that for filters?

**Comment by dahlke** [comment]
You are correct. Oversight on my part. I will implement this and release something next week.

**Comment by dahlke** [comment]
Added the relevant code here: https://github.com/dahlke/terrasnek/commit/3c2642155fc6a4ac13e7f8c1c70777fbcacf5149

**Comment by rorychatterton** [comment]
Great, thanks for the update!

**Comment by dahlke** [comment]
This has been released in [0.1.9](https://github.com/dahlke/terrasnek/releases/tag/0.1.9).
